### PR TITLE
RUELVLA 10-11

### PR DESCRIPTION
**Pull request by MichaelBarcley**
Seeding In-memory Database still not complete, will continue. Other stuff in the two stories are done.
Added a TestController temporarily- it will be deleted when the seeding is complete and it is correct.

### CODE REVIEW

**Comment by MichaelBarcley** [comment]
Stories no10 and no11 finished.

File: rueppellii-lvlup-asp.net-core/Data/SeedInMemoryDatabase.cs [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+                    Id = 3,
+                    Name = "Senior Mentor",
+                    ArchetypeLevels = null
+                }
+            };
+            context.Archetypes.AddRange(archetypes);
+
+            IList<ArchetypeLevel> archetypeLevels = new List<ArchetypeLevel>()
+            {
+
```
Comment (drexeverest) [inline rueppellii-lvlup-asp.net-core/Data/SeedInMemoryDatabase.cs]: ????

File: rueppellii-lvlup-asp.net-core/Models/ArchetypeLevel.cs [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 using System.ComponentModel.DataAnnotations.Schema;
 using System.Linq;
 using System.Threading.Tasks;
 
+
 namespace rueppellii_lvlup_asp.net_core.Models
 {
     public class ArchetypeLevel
     {
+
```
Comment (drexeverest) [inline rueppellii-lvlup-asp.net-core/Models/ArchetypeLevel.cs]: ???

File: rueppellii-lvlup-asp.net-core/Models/User.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
-﻿using System.Collections.Generic;
+﻿using Newtonsoft.Json;
```
Comment (drexeverest) [inline rueppellii-lvlup-asp.net-core/Models/User.cs:1]: Ctrl+R & Ctrl+G

File: rueppellii-lvlup-asp.net-core/Startup.cs [hunk 4]
```diff
@@ -5,37 +5,65 @@
...
 namespace rueppellii_lvlup_asp.net_core
 {
     public class Startup
     {
         public IConfiguration Configuration { get; }
 
-        public Startup(IConfiguration configuration)
+        public Startup(IHostingEnvironment env)
+        {
+            var builder = new ConfigurationBuilder()
```
Comment (drexeverest) [inline rueppellii-lvlup-asp.net-core/Startup.cs:18]: 🤘

File: rueppellii-lvlup-asp.net-core/Startup.cs [hunk 5]
```diff
@@ -5,37 +5,65 @@
...
+                .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+                .AddEnvironmentVariables();
+
+            Configuration = builder.Build();
+        }
+
+        public void ConfigureDevelopmentServices(IServiceCollection services)
+        {
+            services.AddDbContext<LvlUpDbContext>(options =>
+            options.UseInMemoryDatabase("development"));
```
Comment (drexeverest) [inline rueppellii-lvlup-asp.net-core/Startup.cs]: ➡

File: rueppellii-lvlup-asp.net-core/Startup.cs [hunk 6]
```diff
@@ -5,37 +5,65 @@
...
+
+            services.AddMvc();
+        }
+
+        public void ConfigureTestingServices(IServiceCollection services)
         {
-            Configuration = configuration;
+            services.AddDbContext<LvlUpDbContext>(options =>
+            options.UseInMemoryDatabase("testing"));
```
Comment (drexeverest) [inline rueppellii-lvlup-asp.net-core/Startup.cs]: ➡

**Review by drexeverest — APPROVED** [review]
👍

**Pull request merged**